### ISSUE TITLE
Negative Finesse

### ISSUE THREAD

**Issue by alex-ong** [tetrio/issues#268]
### Is your feature request related to a problem? Please describe.
Does Finesse counter have negative values? Some optimizations such as maintaining das from last piece would result in two pieces placed against the wall in 3 keys instead of 4 keys pressed.

Further reading: https://harddrop.com/wiki/DAS_Optimization

### Describe the solution you'd like
It would be ideal to have two counters; standard count, and then negatives totaled. Skillstop at 10-15hz ARR could be used to get them negatives.

### Additional context
I mean if you wanted to be TAS level, you could do A* search for every piece, including DAS charge from previous piece, as well as allowing SkillStop; But all those 0 Finesse people would now jump to 35+ Finesse.

**Comment by alex-ong** [comment]
This is mainly at the high level to compare players who are sub 25, some use pipelining/double-das, das-tapback-cancel-tapback, and would get higher negatives, but still be 0 in the traditional sense.

**Comment by Wirtualnosc** [comment]
To be fair, there's a lot of optimizations that could be done to finesse in first place. Currently, finesse doesn't account for tucks. From what I know, tetresse and jstris handle only simpler scenarios (no tucks, no account for 20G etc), so there's a good chance such improvements wouldn't be easy to implement.

**Comment by alex-ong** [comment]
I agree. But for single piece (no charge maintannce from previous piece) we can at least use a* if it's not too expensive per piece. Though tbf finesse is usually just got sprint right
